samples/kvsWebRTCClientMasterGstreamerSample.c

```diff
@@ -338,6 +338,8 @@ INT32 main(INT32 argc, CHAR* argv[])
     STATUS retStatus = STATUS_SUCCESS;
     PSampleConfiguration pSampleConfiguration = NULL;
 
+    SET_INSTRUMENTED_ALLOCATORS();
```

Comment:
Do we need to set these allocators since we set `-DINSTRUMENTED_ALLOCATORS` to OFF in `CMake`? There is no harm in adding it to the samples, but, just want to confirm.

Reply:
The macros evaluate to no-op if the compile define is not set. The idea here is to control the behavior by not modifying the code but rather providing compile time definition

Reply:
BTW, this is very useful to have. Our unit tests do have it and we caught mem leaks on negative paths whereas the address sanitizer reported a clean run.

Reply:
That's one thing I forgot to mention in the description. I tested with and without the instrumented mem allocators.